persistent/_timestamp.c

```diff
@@ -27,9 +27,40 @@ static char TimeStampModule_doc[] =
 "$Id$\n";
 
 
+
+#define SCONV ((double)60) / ((double)(0x10000)) / ((double)(0x10000))
+#define TS_BASE_YEAR 1900
+#define TS_MINUTES_PER_DAY 1440
+/* We pretend there are always 31 days in a month; this has us using
+   372 days in a year in some calculations */
+#define TS_DAYS_PER_MONTH 31
+#define TS_MONTHS_PER_YEAR 12
+#define TS_MINUTES_PER_MONTH (TS_DAYS_PER_MONTH * TS_MINUTES_PER_DAY)
+#define TS_MINUTES_PER_YEAR (TS_MINUTES_PER_MONTH * TS_MONTHS_PER_YEAR)
+
+#define TS_PACK_UNSIGNED_INTO_BYTES(v, bytes) do { \
```

Comment:
Pack unsigned *what* into bytes?

Reply:
The given *bytes*. I assume you're looking for something like 'INTO_4_BYTES'?

Reply:
What are the semantics of `v`, as we use it?

Reply:
Documentation added.